frontstage/templates/errors/400-error.html

```diff
@@ -1,7 +1,7 @@
 {% import 'partials/section.html' as section %}
 {% extends "layouts/_twocol.html" %}
 
-{% block page_title %}An error has occurred - ONS Business Surveys{% endblock %}
+{% block page_title %}An error has occurred - ONS Business Surveys - 400){% endblock %}
```

Comment:
I'd be potentially inclined to say `4xx` (and `5xx` below) as these'll pop up for a variety of those error types.

Is there anyway the actual code can be pushed into the template? Might be useful to have breakdowns of the _actual_ error codes in analytics?

Reply:
Told not to have any technical jargon in the templates, this is just so we can see which pages are being hit and how often in the analytics. Also fairly sure once it arrives at frontstage they are in fact all 500's or 400's (except specifically 404 which has a different template)

Reply:
Fair enough 👍